CIP-????/README.md

```diff
@@ -0,0 +1,121 @@
+---
+CIP: ?
+Title: Maybe Datum
+Category: Plutus, Ledger
```

Comment:
```suggestion
Category: Plutus
```
The current editorial view is that we should only have one category per CIP: https://github.com/cardano-foundation/CIPs/issues/433#issuecomment-1386614231

and specifically (via precedent of https://github.com/cardano-foundation/CIPs/pull/437) Plutus proposals which also affect the Ledger should be given a category of `Plutus` (unless maybe needing more involvement from the Ledger team): https://github.com/cardano-foundation/CIPs/issues/433#issuecomment-1386885858